testing/export/export_package-install-with-components-and-version.cmake

```diff
@@ -0,0 +1,39 @@
+#=============================================================================
+# Copyright (c) 2018-2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include(${rapids-cmake-dir}/export/package.cmake)
+
+rapids_export_package( build FAKE_PACKAGE test_export_set VERSION 14.7.2 COMPONENTS abc def)
```

Comment:
```suggestion
rapids_export_package( install FAKE_PACKAGE test_export_set VERSION 14.7.2 COMPONENTS abc def)
```

Reply:
I think you need `s/build/install/g` in all of the `export_package-install*` files.